include/fc/io/cfile.hpp

```diff
@@ -94,14 +118,20 @@ class cfile {
    }
 
    void write( const char* d, size_t n ) {
-      size_t result = fwrite( d, 1, n, _file.get() );
+      if(!is_open())
+         throw std::ios_base::failure("cfile is not open");
+
+      size_t result = fwrite( d, WRITE_SIZE, n, _file.get() );
```

Comment:
Where and what is WRITE_SIZE? This needs to be one because `n` is in terms of char.

Reply:
oops- thanks for catching that. i  cleaned up the code with out trying to compile